packages/test/test-drivers/src/odspTestDriver.ts

```diff
@@ -309,7 +309,7 @@ export class OdspTestDriver implements ITestDriver {
             const driveItem = await getDriveItemByRootFileName(
                 this.config.siteUrl,
                 undefined,
-                `/${this.config.directory}/${testId}.fluid`,
+                `/${this.config.directory}/${testId}.loadTest`,
```

Comment:
this is used by more than the load test. could we call it fluidTest? or something like that?

Reply:
Since our logs truncate the first 5 letters fluidTest may be confusing, I can rename it to .tstFluid instead.